python/hsml/core/explicit_provenance.py

```diff
@@ -0,0 +1,338 @@
+#
```

Comment:
PR is missing documentation for provenance links. Stuff like:

https://github.com/logicalclocks/feature-store-api/blob/88e21d3dcf01f545104007858b9fa07329693cb4/auto_doc.py#L254

https://github.com/logicalclocks/feature-store-api/blob/88e21d3dcf01f545104007858b9fa07329693cb4/docs/templates/api/links.md?plain=1

Reply:
added them similar to hsfs